### PR TITLE
Book show 50147

### PR DESCRIPTION
**Pull request by abdullahallmehedi**

### CODE REVIEW

File: app/views/books/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+<% provide :title, @book.title %>
+<div class="row">
+  <div class="col-md-3">
+    <div id="links">
+        <% @book_images.each do |book_image| %>
```
Comment (Rownaknira) [inline app/views/books/show.html.erb]: indent

File: app/views/books/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        <a href="<%= book_image.image.url %>" data-gallery>
```
Comment (Rownaknira) [inline app/views/books/show.html.erb]: indent

File: app/views/books/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            <%= image_tag book_image.image.url(:thumb), class: "img-thumbnail" %>
```
Comment (Rownaknira) [inline app/views/books/show.html.erb]: indent

File: app/views/layouts/_admin_header.html.erb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    <%= t ".categories" %> <span class="caret"></span>
+  <% end %>
+  <ul class="dropdown-menu">
+    <li><%= link_to t(".list"), admin_categories_path %></li>
+    <li><%= link_to t(".add"), new_admin_category_path %></li>
+  </ul>
+</li>
+<li class="dropdown">
+  <%= link_to  "#", class: "dropdown-toggle", data: {toggle: "dropdown"},
+   role: "button", aria: {haspopup: "true", expanded: "false"} do %>
```
Comment (Rownaknira) [inline app/views/layouts/_admin_header.html.erb]: indent

File: app/views/layouts/_header.html.erb [hunk 5]
```diff
@@ -4,25 +4,28 @@
...
-                  class: "btn" %></li>
+            <li><%= link_to t(".home"), current_user %></li>
+          <% if current_user.admin? %>            
+            <%= render "layouts/admin_header" %>
+          <% else %>            
+            <%= render "layouts/user_header" %>
           <% end %>
+          <li class="dropdown">
+            <%= link_to  "#", class: "dropdown-toggle", data: {toggle: "dropdown"},
+             role: "button", aria: {haspopup: "true", expanded: "false"} do %>
```
Comment (Rownaknira) [inline app/views/layouts/_header.html.erb]: indent

File: app/views/layouts/_user_header.html.erb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+<li><%= link_to t(".books"), books_path %></li>
+<li><%= link_to t(".categories"), categories_path %></li>
+<li><%= link_to t(".users"), users_path %></li>
+<li class="dropdown">
+  <%= link_to  "#", class: "dropdown-toggle", data: {toggle: "dropdown"},
+   role: "button", aria: {haspopup: "true", expanded: "false"} do %>
```
Comment (Rownaknira) [inline app/views/layouts/_user_header.html.erb]: indent

**Comment by SSalekin** [comment]
ok

**Comment by Rownaknira** [comment]
ok

File: app/views/books/show.html.erb [hunk 7]
```diff
@@ -1,38 +1,135 @@
...
+          <dd><%= @book.num_of_pages %></dd>
+          <dt><%= t ".category" %></dt>
+          <dd>
+            <%= link_to @book.category.name, category_path(@book.category) %>
+          </dd>
+          <dt><%= t ".rating" %></dt>
+          <dd><div id="rating"></div></dd>
+        </dl>
+        <script type="text/javascript">
+          $(document).ready(function () {
```
Comment (dieunb) [inline app/views/books/show.html.erb]: move to javascript file

File: app/assets/javascripts/books.js [hunk 8]
```diff
@@ -0,0 +1,8 @@
+$(document).ready(function() {
+  $("#book_publish_date").datepicker({format: "dd/mm/yyyy"});
+  $("#rating").raty({
+    readOnly : true,
+    path : "/assets",
+    score : <%= @book.rating %>
```
Comment (dieunb) [inline app/assets/javascripts/books.js]: Are you sure about this ruby code can run?

Comment (abdullahallmehedi) [inline app/assets/javascripts/books.js]: yap. I have tested it. while this code was inside "show.html.erb"

**Pull request merged**